Unable to use Camera via ImagePicker through Expo Go on Android

### Minimal reproducible example

https://github.com/chris-nolan/tunnel-app

### Summary

ImagePicker should load the native camera UI and allow a picture to be taken. This works fine when doing a development build, however when running through Expo Go the following error occurs:

`
 WARN  Possible Unhandled Promise Rejection (id: 0):
Error: Call to function 'ExponentImagePicker.launchCameraAsync' has been rejected.
→ Caused by: java.lang.NullPointerException: Parameter specified as non-null is null: method host.exp.exponent.experience.d.checkSelfPermission, parameter permission
Error: Call to function 'ExponentImagePicker.launchCameraAsync' has been rejected.
→ Caused by: java.lang.NullPointerException: Parameter specified as non-null is null: method host.exp.exponent.experience.d.checkSelfPermission, parameter permission
    at construct (native)
    at apply (native)
    at _construct (http://192.168.68.81:19000/index.ts.bundle?platform=android&dev=true&hot=false:3251:28)
    at Wrapper (http://192.168.68.81:19000/index.ts.bundle?platform=android&dev=true&hot=false:3213:25)
    at construct (native)
    at _createSuperInternal (http://192.168.68.81:19000/index.ts.bundle?platform=android&dev=true&hot=false:112723:322)
    at call (native)
    at CodedError (http://192.168.68.81:19000/index.ts.bundle?platform=android&dev=true&hot=false:112731:26)
`

### Environment


expo-env-info 1.0.5 environment info:
    System:
      OS: Windows 10 10.0.19044
    Binaries:
      Node: 16.13.0 - ~\AppData\Local\Volta\tools\image\node\16.13.0\node.EXE
      Yarn: 1.22.17 - ~\AppData\Local\Volta\tools\image\yarn\1.22.17\bin\yarn.CMD
      npm: 8.1.4 - ~\AppData\Local\Volta\tools\image\npm\8.1.4\bin\npm.CMD
    npmPackages:
      @expo/webpack-config: ^18.0.1 => 18.0.1
      expo: ~48.0.5 => 48.0.5
      react: 18.2.0 => 18.2.0
      react-dom: 18.2.0 => 18.2.0
      react-native: 0.71.3 => 0.71.3
      react-native-web: ~0.18.10 => 0.18.12
    Expo Workflow: managed

--- a/android/expoview/src/main/java/host/exp/exponent/utils/ScopedPermissionsRequester.kt
+++ b/android/expoview/src/main/java/host/exp/exponent/utils/ScopedPermissionsRequester.kt
@@ -38,6 +38,9 @@ class ScopedPermissionsRequester(private val experienceKey: ExperienceKey) {
     permissionsResult = mutableMapOf()
 
     for (permission in permissions) {
+      if (permission == null) {
+        continue
+      }
       val globalStatus = currentActivity.checkSelfPermission(permission)
       if (globalStatus == PackageManager.PERMISSION_DENIED) {
         permissionsToRequestGlobally.add(permission)

